tests/testthat/test-anovarepeatedmeasures.R

```diff
@@ -344,141 +344,6 @@ test_that("Analysis handles errors", {
 
 })
 
-# Order restrictions
-
-test_that("Model Comparison Table results match", {
-  options <- initOpts()
-  options$restrictedModels <- list(list(restrictionSyntax = "DrinkBeer.ImageryPositive == DrinkBeer.ImageryNegative", modelName = "Model 1", 
```

Comment:
why did you delete these tests?

Reply:
The rm anova restrictions and an(c)ova restrictions now use the same functions. For an(c)ova, I also made new tests using the examples on the https://restriktor.org/ website, so the majority of the restrictions code is tested there for consistency with the restriktor package. In the tests for rm anova, I focused on making tests that trigger code that is unique to rm anova.

The rewrite made the input and output structure different, so it was easier and less error prone to make new tests rather than modifying the old ones. I could reinstate these tests and fix them to work with the rewrite, but it seemed like an effort that does not add much.